sql/core/src/test/resources/sql-tests/results/postgreSQL/aggregates_part2.sql.out

```diff
@@ -191,113 +206,113 @@ SELECT
   BOOL_OR(NOT b2)  AS f5,
   BOOL_OR(NOT b3)  AS t6
 FROM bool_test
--- !query 15 schema
-struct<t1:boolean,t2:boolean,f3:boolean,n4:boolean,f5:boolean,t6:boolean>
--- !query 15 output
-true	true	false	NULL	false	true
-
-
--- !query 16
-select min(unique1) from tenk1
 -- !query 16 schema
-struct<min(unique1):int>
+struct<t1:boolean,t2:boolean,f3:boolean,n4:boolean,f5:boolean,t6:boolean>
 -- !query 16 output
-0
+true	true	false	NULL	false	true
 
 
 -- !query 17
-select max(unique1) from tenk1
+select min(unique1) from tenk1
 -- !query 17 schema
-struct<max(unique1):int>
+struct<min(unique1):int>
 -- !query 17 output
-9999
+0
 
 
 -- !query 18
-select max(unique1) from tenk1 where unique1 < 42
+select max(unique1) from tenk1
 -- !query 18 schema
 struct<max(unique1):int>
 -- !query 18 output
-41
+9999
 
 
 -- !query 19
-select max(unique1) from tenk1 where unique1 > 42
+select max(unique1) from tenk1 where unique1 < 42
 -- !query 19 schema
 struct<max(unique1):int>
 -- !query 19 output
-9999
+41
 
 
 -- !query 20
-select max(unique1) from tenk1 where unique1 > 42000
+select max(unique1) from tenk1 where unique1 > 42
 -- !query 20 schema
 struct<max(unique1):int>
 -- !query 20 output
-NULL
+9999
 
 
 -- !query 21
-select max(tenthous) from tenk1 where thousand = 33
+select max(unique1) from tenk1 where unique1 > 42000
 -- !query 21 schema
-struct<max(tenthous):int>
+struct<max(unique1):int>
 -- !query 21 output
-9033
+NULL
 
 
 -- !query 22
-select min(tenthous) from tenk1 where thousand = 33
+select max(tenthous) from tenk1 where thousand = 33
 -- !query 22 schema
-struct<min(tenthous):int>
+struct<max(tenthous):int>
 -- !query 22 output
-33
+9033
 
 
 -- !query 23
-select distinct max(unique2) from tenk1
+select min(tenthous) from tenk1 where thousand = 33
 -- !query 23 schema
-struct<max(unique2):int>
+struct<min(tenthous):int>
 -- !query 23 output
-9999
+33
 
 
 -- !query 24
-select max(unique2) from tenk1 order by 1
+select distinct max(unique2) from tenk1
 -- !query 24 schema
 struct<max(unique2):int>
 -- !query 24 output
 9999
 
 
 -- !query 25
-select max(unique2) from tenk1 order by max(unique2)
+select max(unique2) from tenk1 order by 1
 -- !query 25 schema
 struct<max(unique2):int>
 -- !query 25 output
 9999
 
 
 -- !query 26
-select max(unique2) from tenk1 order by max(unique2)+1
+select max(unique2) from tenk1 order by max(unique2)
 -- !query 26 schema
 struct<max(unique2):int>
 -- !query 26 output
 9999
 
 
 -- !query 27
-select t1.max_unique2, g from (select max(unique2) as max_unique2 FROM tenk1) t1 LATERAL VIEW explode(array(1,2,3)) t2 AS g order by g desc
+select max(unique2) from tenk1 order by max(unique2)+1
```

Comment:
Just to be clear, these aren't real differences, just how github is interpreting the diff?

Reply:
there are not real diff